src/modules/template-details/TemplateDetails.tsx

```diff
@@ -56,105 +59,129 @@ const TemplateDetails = ({ match }: TemplateDetails.Props) => {
     setConfirmDeleteOpen(false);
   }, []);
 
+  const forkTemplateHandler = async () => {
```

Comment:
Please use one convention via whole project - handleFork will be ok

Reply:
done